include/mcptam/Tracker.h

```diff
@@ -83,11 +84,16 @@
 #include <ros/ros.h>
 #include <map>
 #include <string>
+#include <iostream>
```

Comment:
do we need these? should only be using ROS messaging.  Are we writing to a file?